lib/worker/batcher.ex

```diff
@@ -524,16 +524,17 @@ defmodule BorsNG.Worker.Batcher do
     |> Patch.all_for_batch()
     |> Repo.all()
 
+    send_message(repo_conn, patches, {:succeeded, statuses})
+
     if toml.use_squash_merge do
       Enum.each(patches, fn patch ->
+        GitHub.post_comment!(repo_conn, patch.pr_xref, "# Pull request successfully merged into master.")
```

Comment:
It doesn't look like you properly rebased this onto https://github.com/bors-ng/bors-ng/pull/823

Reply:
👍 fixing now